src/main/java/au/org/ala/biocache/dto/DownloadDoiDTO.java

```diff
@@ -32,6 +39,9 @@
     private long recordCount;
     private String requestTime;
     private String queryTitle;
+    private String displayTemplate;
+
+    Map<String, String> applicationMetadata;
```

Comment:
Merge conflict issue was not resolved properly here as this field is duplicated. It should also be private rather than package-private.